aya/src/programs/uprobe.rs

```diff
@@ -121,6 +122,50 @@ impl UProbe {
 
         attach(&mut self.data, self.kind, &path, sym_offset + offset, pid)
     }
+
+    /// Attaches the program to an address within userspace process' own
+    /// address space.
+    ///
+    /// Attaches the uprobe to the given address defined in the `addr`
+    /// argument. If `pid` is not `None`, the program executes only when the target
+    /// function is executed by the given `pid`.
+    ///
+    /// If the program is an `uprobe`, it is attached to the *start* address of the target
+    /// function.  Instead if the program is an `uretprobe`, it is attached to the return address of
+    /// the target function.
+    ///
+    /// # Examples
+    ///
+    /// In a separate crate (let's assume it's a `my-uprobes` crate) in
+    /// `lib.rs`:
+    ///
+    /// ```no_run
+    /// #[no_mangle]
+    /// #[inline(never)]
+    /// pub extern "C" fn my_function(_retp: *mut i32, _val: i32) {}
+    /// ```
+    ///
+    /// Main code:
+    ///
+    /// ```no_run
+    /// # let mut bpf = Bpf::load_file("ebpf_programs.o")
```

Comment:
Missing a semicolon, causing CI to fail.

Reply:
Shame on me, I have to make running `cargo doc` my routine.

Reply:
Fixed